src/app/static/src/js/components/runReport/runReport.vue

```diff
@@ -81,6 +82,32 @@
             }
         },
         methods: {
+            refreshGit: function () {
```

Comment:
This method needs to trigger an update to `gitBranches`, which is initially inherited from the Freemarker template (see also comment below). Currently if I create a new branch and click "Refresh git" I don't see the branch.